examples/FFT-examples.ipynb

```diff
@@ -8,6 +8,21 @@
     "The following is a brief demonstration of the FFT forecasting model. This model is especially suited for data that is very seasonal. The datasets chosen for this demonstration were selected accordingly."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# fix pythonpath when working locally\n",
+    "from os.path import dirname, basename\n",
+    "import sys\n",
+    "\n",
+    "cwd = os.getcwd()\n",
+    "if basename(cwd) == 'examples':\n",
+    "    sys.path.insert(0, dirname(cwd))"
+   ]
```

Comment:
Maybe it could be put as a separate function common to all notebooks (in separate file maybe). Plus setting this path could be configurable by env variable for examples 'local-develop' which you could set locally. What do you think about it?